### PR TITLE
Feature/ffxiv weather

### PR DESCRIPTION
**Pull request by mRmetr0**

### CODE REVIEW

File: src/extensions/ffxiv_weather.py [hunk 1]
```diff
@@ -0,0 +1,67 @@
+from discord.ext import commands
+import discord
+
+import datetime
+import ffxivweather
+
+list = ["Limsa Lominsa", "Middle La Noscea", "Lower La Noscea",
```
Comment (MythicalSora) [inline src/extensions/ffxiv_weather.py:7]: This list is only limited to zones within Limsa Lominsa (which is 1/8th of the available zones). We should include every existing region.

File: src/extensions/ffxiv_weather.py [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+        return embed
+
+    @commands.slash_command(
+        guild_ids=[1033025456242434089],
+        description="Get the weather forecast of any place in FFXIV"
+    )
+    async def ffxivweather(
+        self,
+        ctx: discord.ApplicationContext,
+        zone: discord.Option(
```
Comment (MythicalSora) [inline src/extensions/ffxiv_weather.py:59]: Make this a standard input field, we can't put all available subzones within the Discord list limit

File: src/extensions/ffxiv_weather.py [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+list = ["Limsa Lominsa", "Middle La Noscea", "Lower La Noscea", 
+    "Eastern La Noscea", "Western La Noscea", "Upper La Noscea", 
+    "Outer La Noscea", "Mist"]
+
+
+class FFXIVWeather(commands.Cog):
+    def __init__(self, bot):
+        self.client = bot
+
+    def __create_embed(self, zone, amount: int = 15):
```
Comment (MythicalSora) [inline src/extensions/ffxiv_weather.py:16]: This function is named inaccurately and does too much on it's own. You should split this code up into at least 2 or 3 separately named functions.

**Review by ameijboom — CHANGES_REQUESTED** [review]
Please look through the provided comments.

**Review by ameijboom — APPROVED** [review]
Lgtm

**Pull request merged**